test/behavior/zon/escaped_struct.zon

```diff
@@ -0,0 +1,2 @@
+// zig fmt: off
```

Comment:
Why disable formatting here?

Reply:
The formatter removes the escape around foo since they aren't necessary, but I wanted to test what the error messages look like when there are unnecessary escapes